ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -138,6 +268,12 @@ <h2 id="timer-text" class="show-expression"></h2>
 
     }
   });
+  $("#next").click(() => {
+    console.log("next problem clicked");
+    if(curr_problem_index < problems.length - 1 && !$("#next").hasClass("disabled")){
```

Comment:
This should have parens around the left expressions being evaluated by the && operator to make it a bit more readable: `(curr_problem_index < problems.length - 1) && !$("#next").hasClass("disabled")`